src/credentials/examples/DeviceAttestationVerifierExample.cpp

```diff
@@ -209,22 +213,24 @@ AttestationVerificationResult ExampleDACVerifier::VerifyAttestationInformation(c
                                                                                const ByteSpan & dacCertDerBuffer,
                                                                                const ByteSpan & attestationNonce)
 {
+    VendorId dacVendorId = VendorId::NotSpecified;
     // match DAC and PAI VIDs
     if (!paiCertDerBuffer.empty())
```

Comment:
Issue in the existing code: PAI certificate is not option but mandatory so this `if` statement is not needed.
Might be a good idea in the beginning of this function to Verify that all input paramagnets are NOT empty spans.

Reply:
Addressed in 92dd5d5. It will shorten the review cycle if we can get all the review comments together.